FireRender.Maya.OSX/frMayaPluginMac/frMayaMac/RadeonProRender.xcodeproj/project.pbxproj

```diff
@@ -1498,7 +1855,6 @@
 		08FB7795FE84155DC02AAC07 /* Source */ = {
 			isa = PBXGroup;
 			children = (
-				F1ADFC0029F2B012006A605E /* VulcanUtils.h */,
```

Comment:
When I try to build this branch I have build failures due to search paths for header files on OSX.
I was able to make some progress by adding search paths but I am not a programmer so I will just wait until an official release comes out as I do not know what I am doing well enough.
I have the dependencies installed per the readmeOSX and the process works on the stable branch.
Thank you for your many contributions to the project.